src/app/personside/dialogpanel/sendMelding/standardTekster/StandardTekster.tsx

```diff
@@ -18,21 +17,26 @@ import { useFetchWithLog } from '../../../../../utils/hooks/useFetchWithLog';
 import { loggEvent } from '../../../../../utils/frontendLogger';
 import { useErKontaktsenter } from '../../../../../utils/enheterUtils';
 import { useRestResource } from '../../../../../rest/consumer/useRestResource';
+import LazySpinner from '../../../../../components/LazySpinner';
+import { guid } from 'nav-frontend-js-utils';
 
 interface Props {
     appendTekst(tekst: string): void;
 }
 
-const FormContainer = styled.form`
+const StyledForm = styled.form`
     height: 100%;
     display: flex;
     flex-direction: column;
+    background-color: white;
```

Comment:
Gjør denne noe? Bakgrunnfargen blir vel uansett overskrevet av `SokefeltStyledNav`

Reply:
Gjør ikke så mye nytte for seg i modalen som har hvit bakgrunn, men satt den opp i standalone-komponenter, og der var bakgrunnen grå og derfor litt vanskelig å jobbe med. Så nei og ja :-)